python/lsst/ip/diffim/utils.py

```diff
@@ -820,14 +820,40 @@ def fitDipoleSource(self, source, **kwds):
         fitResult = alg.fitDipole(source, **kwds)
         return fitResult
 
-    def detectDipoleSources(self, doMerge=True, diffim=None, detectSigma=5.5, grow=3):
+    def detectDipoleSources(self, doMerge=True, diffim=None, detectSigma=5.5, grow=3, weighting=True,
+                            minBinSize=32):
         """!Utility function for detecting dipoles.
 
         Detect pos/neg sources in the diffim, then merge them. A
         bigger "grow" parameter leads to a larger footprint which
         helps with dipole measurement for faint dipoles.
-        """
 
+        Parameters
+        ----------
+        doMerge : `bool`
+           Wether to merge the positive and negagive detections into a single source table
+        diffim : `lsst.afw.image.exposure.exposure.ExposureF`
+           Difference image on which to perform detection
+        detectSigma : `float`
+           Threshold for object detection
+        grow : `int`
+           Number of pixels to grow the footprints before merging
+        weighting : `bool`
+           Wether to use weighting in the background (re)estimation (only applies is useApprox=True)
```

Comment:
What is `useApprox`? I'd naïvely expect it to be another argument to `detectDipoleSources`, but it isn't.

(In fact, I bet I can find it buried in the detection task config, but that's not clear from this docstring.)

Reply:
Sorry, that's a background config:
https://github.com/lsst/meas_algorithms/blob/master/python/lsst/meas/algorithms/subtractBackground.py#L98
        doc="Use Approximate (Chebyshev) to model background."
This might be clearer (at least tells you where to look without being too verbose)?:
(only applies if detectConfig.background.useApprox=True)